Proton/Sources/Swift/Editor/EditorView.swift

```diff
@@ -233,6 +234,7 @@ open class EditorView: UIView {
         set { richTextView.contentInset = newValue }
     }
 
+    @available(iOS 11.1, *)
     @available(iOSApplicationExtension 11.1, *)
```

Comment:
Are these two equivalent?

Reply:
this came up as Autofix suggestion. I wasn't 100% sure, so added it alongside. Might change it later if these turn out to be same.